hera_cal/io.py

```diff
@@ -1292,7 +1294,7 @@ def load_cal(input_cal, return_meta=False):
 
 def write_cal(fname, gains, freqs, times, flags=None, quality=None, total_qual=None, write_file=True,
               return_uvc=True, outdir='./', overwrite=False, gain_convention='divide',
-              history=' ', x_orientation="east", telescope_name='HERA', cal_style='redundant',
+              history=' ', x_orientation="north", telescope_name='HERA', cal_style='redundant',
```

Comment:
I don't remember where we landed on this, so I just want to clarify - should writing error if we don't actually know the `x_orientation`?

Reply:
So, I've changed the default to match H3C. However, I took out a change I had originally made, which was to turn `None` into `"north"`. So if the `x_orientation` is `None`, usually because it was inherited from some data file, this will throw an error because x_orientation is a required UVCal parameter.

Reply:
so it sounds like there's an assumption that `x_orientation` will always be set when running `write_cal` -- either a known orientation, or `None`?

Reply:
Write_cal is meant to use all default parameters unless you set them, so it's easy to make a brand-new pyvudata compatible file. 

However, everytime write_cal is called in the pipeline, I bring in the x_orientation from the associated data file. If the x_orientation is not defined (i.e. `None`) and you use a routine like redcal, this will raise an error, which I think is the desired behavior.

Reply:
Ok, I think that's reasonable enough.